libs/openchallenges/home/src/lib/home.component.ts

```diff
@@ -50,6 +51,7 @@ export class HomeComponent {
     this.privacyPolicyUrl = this.configService.config.privacyPolicyUrl;
     this.termsOfUseUrl = this.configService.config.termsOfUseUrl;
     this.apiDocsUrl = this.configService.config.apiDocsUrl;
+    this.showAnnouncement = this.configService.config.showAnnouncement === 'true';
```

Comment:
@tschaffter I was just testing if it works in docker, but I found the env variables we defined in the `.env/config.json` will result in string 'false', and in typescript, strings like "false" are truthy, like below:
```typescript
if ("false") {
  // it will be truthy and run
}
```
I just converted the config variable to boolean here for now, but feel free to let me know if there's a better solution.

Reply:
Looks good. Feel free to request another review when ready.